src/app/users/users-table.component.ts

```diff
@@ -41,7 +41,7 @@ export class TableState {
 })
 export class UsersTableComponent implements OnInit, OnDestroy, AfterViewInit, OnChanges {
 
-  @Input() displayedColumns = [ 'select', 'profile', 'name', 'visitCount', 'joinDate', 'lastLogin', 'roles', 'action' ];
+  @Input() displayedColumns = [ 'select', 'profile', 'name', 'visitCount', 'joinDate', 'lastLogin', 'roles' ];
```

Comment:
should be based on condition

Reply:
@lmmrssa 'action' is an element in `displayedColumns`, how to exclude 'action' based on condition. Is that `{action:isDialog}`

Reply:
@lmmrssa I switch the status of `displayedColumns` in `user-table.component.ts`. I think it is working.